src/sentry/notifications/notifications/rules.py

```diff
@@ -159,6 +162,19 @@ def get_context(self) -> MutableMapping[str, Any]:
         if not enhanced_privacy:
             context.update({"tags": self.event.tags, "interfaces": get_interface_list(self.event)})
 
+        has_session_replay = features.has("organizations:session-replay", self.organization)
+        has_replay_link = features.has(
+            "organizations:session-replay-issue-emails", self.organization
+        )
+        replay_id = get_replay_id(self.event)
+        if has_session_replay and has_replay_link and replay_id:
+            context.update(
+                {
+                    "replay_id": replay_id,
+                    "replay_url": get_replay_details_link(self.organization, replay_id),
```

Comment:
@ryan953 could you add a test on the context?

Reply:
np